schema/workflow.json

```diff
@@ -193,14 +193,9 @@
     "error": {
       "type": "object",
       "properties": {
-        "error": {
```

Comment:
I would not remove this, but rename it to "name" so users can still have a domain-specific name that they can use for this error. 
so replace with "name" property type string with description "Domain-specific error name"